redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/incidentImageCarousel.test.js

```diff
@@ -19,4 +32,24 @@ describe('c-incident-image-carousel', () => {
         const labelElement = element.shadowRoot.querySelector('p');
         expect(labelElement.textContent).toContain('no images');
```

Comment:
Should be equal comparison for `There are currently no images of damage for this case.`